dtfs-central-api/src/services/changeStream/changeStreamApi.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import { ChangeStreamDocument } from 'mongodb';
+import { InvalidEnvironmentVariableError } from '../../errors/invalid-environment-variable.error';
+
+/**
+ * Checks whether the document is a document that needs to be sent to the audit API
+ * and sends it if it is
+ * @param changeStreamDocument Change Stream Document event from mongodb API
+ * @returns
+ */
+export const postAuditDetails: (changeStreamDocument: ChangeStreamDocument) => Promise<void> = async (changeStreamDocument: ChangeStreamDocument) => {
```

Comment:
Intentionally not wrapped in try-catch so that the calling function can handle any errors thrown :) The Promise<void> should be right, it doesn't return anything it's just async